### DISCUSSION TITLE
Are there plans to become independent of MUI?

### DISCUSSION THREAD

**Discussion by eduwesley** [TexteaInc/json-viewer#474] [Q&A]
Currently, the project requires the installation of [Material-UI](https://mui.com/). However, this poses a problem if you are already using another component library or if you are not using any, as conflicts in styling or even just the size of the library can arise.

Are there plans to not use the Material UI library? Or is this not an option?

**Comment by pionxzh** [comment]
We do have a plan to drop MUI since it's way too heavy for a json viewer. But I couldn't give a promise on the timeline based on my current workload. Let me know if you want to contribute to this migration.

**Reply by eduwesley** [reply]
I would be happy to help

**Reply by pionxzh** [reply]
Sure. Let me know if you need any help. 👍

**Reply by pionxzh** [reply]
@eduwesley I have opened #515 as the main thead of V5 which will be the version without MUI. Let me know if you are still interesting on this. 🙏
